apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/Button/Button.tsx

```diff
@@ -0,0 +1,78 @@
+import { ReactElement } from 'react'
+import { TreeBlock } from '@core/journeys/ui'
+import ViewDayOutlinedIcon from '@mui/icons-material/ViewDayOutlined'
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined'
+import FormatShapesRoundedIcon from '@mui/icons-material/FormatShapesRounded'
+import LinkRoundedIcon from '@mui/icons-material/LinkRounded'
+import capitalize from 'lodash/capitalize'
+import { GetJourney_journey_blocks_ButtonBlock as ButtonBlock } from '../../../../../../../__generated__/GetJourney'
+import { Attribute } from '../..'
+import { ColorDisplayIcon } from '../../../ColorDisplayIcon'
+
+export function Button({
+  id,
+  buttonVariant,
+  buttonColor,
+  size,
+  startIcon,
+  endIcon,
+  action
+}: TreeBlock<ButtonBlock>): ReactElement {
+  return (
+    <>
+      <Attribute
+        id={`${id}-link-action`}
```

Comment:
```suggestion
        id={`${id}-button-action`}
```
Add `button` to all these id's and change id-font-variant in Typography.ts